text/0000-volta-updates.md

```diff
@@ -0,0 +1,114 @@
+[critique]: #critique
+
+Earlier iterations of this RFC treated the update process mostly the same between Managed and Individual installs, however when discussing the overall design of the system to support both situations, it became clear that the two are fundamentally disconnected and will need to be treated separately. From there, it grew into the current proposal, where we support both modes and then can incrementally work towards a situation where the Individual users are updated automatically, resulting in them being functionally similar to Managed users.
+
+## Incremental or Monolithic
+
+One question is whether we should tackle the Individual update story incrementally, with the notifier followed by the automatic updater later on. Tackling it all at once would mean that we can present users with a polished, delightful update process from the first time it available. However, the full process will likely take a lot of effort to implement and in the interim there wouldn't be anything. Doing incremental updates will probably take more effort overall, but will allow us to give users a working experience and then progressively enhance it until we are at the final state. It's also likely that we will discover edge cases and workflow issues along the way that will inform our final design for the update experience, making it that much better when we reach the goal.
+
+## New Command Name
+
+Another area of critique is in the name of the new command. `volta init` Seems to fit the use-case cleanly, since it's something you will likely run once and then never need to run again, but there are other potential options:
+
+- `volta bootstrap`
+- `volta start`
+- `volta activate` (Has a clear collision with the existing command that is proposed to be removed)
+- `volta selfinstall`
+- `volta self install`
+
+## Detecting Environment Type
+
+The current proposal detects the user's Environment using the location of the binary relative to `VOLTA_HOME`, however that would preclude us from using an Individual installer (with automatic updates) that uses a directory other than `VOLTA_HOME` for the binary files. Is there a different way we can detect which environment the user is working under?
+
+## Windows
+
+The Windows automatic update story is complicated, since the binaries are currently installed into the `Program Files` directory, and the user may or may not have edit access to that directory. Should we update the installer to unpack the files into the `LocalAppData` directory and then run `volta init`, in the same way that the Unix install works? That would guarantee that the user has permissions to the binaries for automatic updates, and would align the Windows and Unix stories more closely.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+- Are there other update models we should implement along the way?
+- Do we want to provide separate installers? One for the "Managed environment" case and one for the "individual user" case?
```

Comment:
We probably don't need a robust answer on this until and unless we have more corporate users. But maybe a simple thing we could offer for now would be tarballs/zips of the binaries, as the simplest possible thing that an IT department could use to automate the retrieval of all the Volta binaries.